ui-components/src/overlays/Modal.scss

```diff
@@ -9,6 +9,8 @@
   @apply rounded-md;
   @apply shadow-md;
   min-width: 25rem;
+  max-height: 80vh;
+  overflow-y: auto;
```

Comment:
Is it possible to override these styles in our `sites/public/styles` directory? Instead of in the ui-components.

Reply:
Ah nice, I'm learning about where our CSS overrides live :)